bears/python/PEP8Bear.py

```diff
@@ -16,6 +16,7 @@ class PEP8Bear(LocalBear):
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     CAN_FIX = {'Formatting'}
+    ASCIINEMA_URL = 'https://asciinema.org/a/163081'
```

Comment:
Looks good. But is it just me on my browser, or is the width too small of the asciinema? The text is cut-off for me :/